docs/pages/database-access/reference/configuration.mdx

```diff
@@ -20,8 +20,10 @@ db_service:
 
   # Matchers for registering AWS-hosted databases.
   aws:
-    # Database types, only "rds" is supported currently.
-  - types: ["rds"]
+    # Database types. Valid options are:
+    # 'rds' - discovers and registers AWS RDS and Aurora databases.
+    # 'redshift' - discovers and registers AWS Redshift databases.
+  - types: ["rds", "redshift"]
```

Comment:
Can you also update configuration reference that's shown on https://goteleport.com/docs/setup/reference/config/?